core/email_api.php

```diff
@@ -1309,8 +1342,14 @@ function email_bug_reminder( $p_recipients, $p_bug_id, $p_message ) {
 			$t_sender_email = '';
 		}
 		$t_header = "\n" . lang_get( 'on_date' ) . ' ' . $t_date . ', ' . $t_sender . ' ' . $t_sender_email . lang_get( 'sent_you_this_reminder_about' ) . ': ' . "\n\n";
-		$t_contents = $t_header . string_get_bug_view_url_with_fqdn( $p_bug_id ) . " \n\n" . $p_message;
 
+/* Kimberly Keown suggested revision to add a custom email signature. (See Line 522.)
+	Revised Code:*/
+	$t_contents = $t_header . string_get_bug_view_url_with_fqdn( $p_bug_id ) . "\n\n" . $p_ message .  "\n\n" . lang_get( 'email_custom_signature' );
```

Comment:
The reason for my earlier comment about testing, is that your code contains a syntax error at the line I pointed out, [as shown in the Travis job log](https://travis-ci.org/mantisbt/mantisbt/jobs/163299393#L201):

```
Parse error: syntax error, unexpected 'message' (T_STRING) in /home/travis/build/mantisbt/mantisbt/core/email_api.php on line 1348
```

(you have a space between `$p_` and `message`).